game/screens/gameplay/controller.py

```
@@ -91,6 +98,15 @@ async def run(self):
                         self.paused = False
                         self._bus.emit('game_resumed', {'now': pygame.time.get_ticks()})
 
+                    if menu_action == "credits":
+                        self._set_paused(True)
+                        from game.screens.credits import CreditsScreen
+                        credits = CreditsScreen(self.screen)
```

Ruff (0.15.9)

[error] Variable `credits` is shadowing a Python builtin

(A001)